rules-tests/TypeDeclaration/Rector/ClassMethod/ReturnUnionTypeRector/Fixture/false_bool_docblock.php.inc

```diff
@@ -26,7 +26,7 @@ namespace Rector\Tests\TypeDeclaration\Rector\ClassMethod\ReturnUnionTypeRector\
 final class FalseBoolDocblock
 {
     /**
-     * @return array|false some description
+     * @return array|bool some description
```

Comment:
new rule to update sub type by docblock seems needed, to ensure not introduce phpstan notice

https://phpstan.org/r/49cbf30f-03e8-4b82-b47e-b649cadc7a34

Reply:
Docblock changes are out of our scope since Rector 0.15. This will have to be dealt manualy.

Reply:
That will ok for 1 or 2 files, but for many files, imo, that's worth it, I think `coding-style` or `code-quality` set will match that

Reply:
That's beyond our scope. I'd leave that for manual review or custom set 👍

Reply:
Also, this seems rather rare case.